README.md

```diff
@@ -40,27 +40,34 @@ The buildkite annotation context to use. Useful to differentiate multiple runs o
 
 Default: `-(.*).xml`
 
-The regular expression (with capture group) that matches the job UUID in the junit file names. This is used to create the job links in the annotation. 
+The regular expression (with capture group) that matches the job UUID in the junit file names. This is used to create the job links in the annotation.
 
 To use this, configure your test reporter to embed the `$BUILDKITE_JOB_ID` environment variable into your junit file names. For example `"junit-buildkite-job-$BUILDKITE_JOB_ID.xml"`.
 
 ### `failure-format` (optional)
 
+Default: `classname`
+
 This setting controls the format of your failed test in the main annotation summary.
 
 There are two options for this:
-* `classname` (the default)
-  * displays: `MyClass::UnderTest text of the failed expectation in path.to.my_class.under_test`
-* `file`
-  * displays: `MyClass::UnderTest text of the failed expectation in path/to/my_class/under_test.file_ext`
 
-### `fail-build-on-error` (optional)  
+- `classname`
+  - displays: `MyClass::UnderTest text of the failed expectation in path.to.my_class.under_test`
+- `file`
+  - displays: `MyClass::UnderTest text of the failed expectation in path/to/my_class/under_test.file_ext`
+
+### `fail-build-on-error` (optional)
 
 Default: `false`
 
-If this setting is true and any errors are found in the JUnit XML files during parsing, the annotation step will exit with a non-zero value, which should cause the build to fail.
+If this setting is true and any errors are found in the JUnit XML files during
+parsing, the annotation step will exit with a non-zero value, which should cause
+the build to fail.
 
-### `min-tests` (optional, integer)
+### `context` (optional)
+
+Default: `junit`
```

Comment:
the conflicts on this file definitely backfired as this no longer makes sense and it appears as if the changes were those from before this commit https://github.com/buildkite-plugins/junit-annotate-buildkite-plugin/commit/17335432822952f0979b90ec3d9fe7ece5cbb743